src/coordinate3d.hpp

```diff
@@ -8,12 +8,11 @@
 #define COORDINATE3D_HPP
 
 class Coordinate3D {
```

Comment:
Just make this a struct.

Reply:
As we program in C++, and not in C, I think the use of ```class``` would be more appropriate, instead of using ```struct```